templates/openstack/consulting.html

```diff
@@ -1,389 +1,493 @@
+            alt="",
+            width="130",
+            height="47",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/b41ec3f5-SBI-Bits_logo.png",
+            alt="",
+            width="615",
+            height="187",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/5af38159-etisalat_logo.svg",
+            alt="",
+            width="108",
+            height="41",
+            hi_def=True,
+            loading="sudlazy"
+            ) | safe
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/f7651dc9-Logo-GARR-300dpi.png",
+            alt="",
+            width="2417",
+            height="917",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
       </div>
     </div>
-    <div class="u-fixed-width">
-      <h3>From zero to private cloud &mdash; OpenStack delivery on rails</h3>
-      <p>With many years experience deploying, supporting, running and upgrading OpenStack clouds, Canonical offers a unique fixed-price private cloud delivery service with a proven reference architecture, called Private Cloud Build.</p>
-    </div>
-    <div class="u-fixed-width u-vertically-spaced">
-      <h3 class="p-heading--four">What's available with Private Cloud Build?</h3>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>OpenStack implementation challenges</h2>
+      <p>Designing an optimal architecture for OpenStack implementation that maximises price-performance requires expert knowledge and years of experience.</p>
+      <p>Here are some common questions one has to ask when implementing OpenStack:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">How to configure the hardware to meet performance and capacity requirements, while not exceeding the budget?</li>
+        <li class="p-list__item is-ticked">Which OpenStack architecture to choose to maximise resource consumption and cost-efficiency of the cloud?</li>
+        <li class="p-list__item is-ticked">Which hypervisor, software-defined networking (SDN) and storage platform to choose?</li>
+        <li class="p-list__item is-ticked">How to design the cloud to meet local security and compliance regulations?</li>
+        <li class="p-list__item is-ticked">How to migrate the existing workloads from VMware or other cloud environments?</li>
+      </ul>
+      <p>Canonical’s private cloud consulting services solve all those challenges, providing a comprehensive implementation package for Charmed OpenStack at a fixed price.</p>
```

Comment:
Can you convert to a straight quote and allow smart quotes module to convert them.